content/docs/identity-providers/cognito.mdx

```diff
@@ -11,9 +11,17 @@ import TabItem from '@theme/TabItem';
 
 This document describes the use of AWS Cognito as an identity provider with Pomerium. It assumes you have already [installed Pomerium](/docs/install/)
 
+:::caution
+While we do our best to keep our documentation up to date, changes to third-party systems are outside our control. Refer to [Quickstart: Register an application with the Microsoft identity platform](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app) from Microsoft's documentation as needed, or [let us know](https://github.com/pomerium/pomerium/issues/new?assignees=&labels=&template=bug_report.md) if we need to re-visit this page.
```

Comment:
I think this was supposed to point at Amazons docs, not azure?

Reply:
Yup, I copied it from another doc and forgot that it has IdP-specific language. Patch incoming.